helm/aws-ebs-csi-driver-app/templates/storage-class.yaml

```diff
@@ -7,5 +7,5 @@ volumeBindingMode: WaitForFirstConsumer
 allowVolumeExpansion: {{ .Values.enableVolumeScheduling }}
 reclaimPolicy: Delete
 parameters:
-  type: gp2
+  type: gp3
```

Comment:
is this supported across all regions including china?

That could be a deal breaker if it's not in China

Reply:
Good thought, I checked that and it's already present in CN

![image](https://user-images.githubusercontent.com/1936982/102499269-418f5680-407b-11eb-8835-754e1b46f990.png)